src/odemis/driver/hamamatsurx.py

```diff
@@ -1322,7 +1342,9 @@ def GenParamSet(self, parameter, value):
                     ShowDelay1Control: Shows or hides the Delay1 status/control dialog
                     ShowDelay2Control: Shows or hides the Delay2 status/control dialog
                     ShowSpectrControl: Shows or hides the Spectrograph status/control dialog
-        :parameter value: (str) TODO which values?? 0 and 1??."""
+        :parameter value: (str) PARAM_TYPE_BOOL."""
+        if not isinstance(value, str):
```

Comment:
You can drop the if, and just always do `str(value)`, as it works also fine if value is already a string.